apps/native-component-list/src/screens/WebBrowserScreen.tsx

```diff
@@ -88,8 +89,10 @@ export default class WebBrowserScreen extends React.Component<{}, State> {
   };
 
   startAuthAsync = async (shouldPrompt: boolean): Promise<any> => {
-    const url = Platform.select({ web: window.location.origin, default: Constants.linkingUrl });
-    const redirectUrl = `${url}/redirect`;
+    const redirectUrl = Platform.select({
+      web: `${window.location.origin}/redirect`,
+      default: Linking.makeUrl('redirect'),
```

Comment:
I think this can just be `Linking.makeUrl()` now.

Reply:
whoops, missed this before merging but addressed in d721e498a6934fad6fb9c7bb3bdb6d7827fff9ad